modules/standard/IO.chpl

```diff
@@ -5229,6 +5229,149 @@ proc _channel.writeBinary(arg:numeric, endian:ioendian) throws {
   }
 }
 
+/*
+   Write a :type:`~String.string` to the channel in binary format
```

Comment:
Should this say "fileWriter" instead?  Channel is the old name and we're using `_channel` so that we can maintain its functionality while supporting the new `fileWriter` type as well

Reply:
Thinking about it more, I think we probably want to define this on the `fileWriter` type itself to start rather than `_channel` for that reason.

Because `fileWriter` is an alias for a particular instantiation of `_channel`, I think this will still allow it to be used with the deprecated type (so the outcome is mostly the same), but it will mean less work when we finally remove the old one and it will mean no one will accidentally be able to write `fileReader.writeBinary` (which I think is technically possible with this route).